apps/librelingo_yaml_loader/librelingo_yaml_loader/yaml_loader.py

```diff
@@ -316,7 +317,7 @@ def _load_module(path, course):
             'Module file "{}" needs to have module name'.format(filepath)
         )
 
-    return Module(title=title, skills=_load_skills(path, skills, course))
+    return Module(title=title, filename=str(path)[len(course.course_dir)+1:], skills=_load_skills(path, skills, course))
```

Comment:
why `str(path)` instead of just `path`?

Reply:
https://github.com/kantord/LibreLingo/pull/1884/files#r774698354

Reply:
Becasue I get `TypeError: 'PosixPath' object is not subscriptable`

Reply:
with relpath this hopefully won't happen, right? I actually never used that in Python though